demo/DEMO.md

```diff
@@ -0,0 +1,51 @@
+# Demo
+
+## Install
+[branch of go-tuf](https://github.com/mnm678/go-tuf/tree/tuf-notary-demo)
+
+## Push to registry
+### create net-monitory image, upload to localhost
+```
+docker run -d -p 5000:5000 ghcr.io/oras-project/registry:v0.0.3-alpha
+docker build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks/net-monitor.git#main
+docker push localhost:5000/net-monitor:v1
+```
+
+### sign with go-tuf library
+```
+tuf init
+tuf add <digest>
+tuf snapshot
+tuf timestamp
+tuf commit
+```
```

Comment:
From earlier discussions notation got feedback and collapsed the commands as much as possible focusing on the first time user. Doesn't mean we don't have these broken down option and once folks are able to start using it they quickly move on to the more complex workflows. 
for e.g. `tuf sign  -o targets.json [REF_DIGEST]` which can be then uploaded using other tools.

Reply:
I agree. This is using the existing go-tuf cli for the initial demo, but in the long term I think this could be simplified, and possibly even combined with the upload command. I'll make another pr with my vision for the overall API.